OpenRA.Mods.Common/Traits/Buildings/Refinery.cs

```diff
@@ -194,6 +217,8 @@ void INotifyCapture.OnCapture(Actor self, Actor captor, Player oldOwner, Player
 
 				// Relink to this refinery
 				dockedHarv.Trait<Harvester>().LinkProc(self);
+
+				UpdateCondition(self);
```

Comment:
`var enabled = dockedHarv != null;` shouldn't change here, so what is the idea behind calling this?

Reply:
You're correct